src/main/resources/db/migration/0200/V246__ModulesDeliverables.sql

```diff
@@ -0,0 +1,108 @@
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+CREATE TABLE accelerator.cohort_modules (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    cohort_id BIGINT NOT NULL REFERENCES accelerator.cohorts,
+    module_id BIGINT NOT NULL REFERENCES accelerator.modules,
+    start_date DATE NOT NULL,
+    end_date DATE NOT NULL
+);
+
+CREATE INDEX ON accelerator.cohort_modules (cohort_id, start_date);
+
+CREATE TABLE accelerator.deliverables (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    deliverable_category_id INTEGER NOT NULL REFERENCES accelerator.deliverable_categories,
+    deliverable_type_id INTEGER NOT NULL REFERENCES accelerator.deliverable_types,
+    module_id BIGINT NOT NULL REFERENCES accelerator.modules,
+    position INTEGER NOT NULL,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    name TEXT NOT NULL,
+    subtitle TEXT,
+    description_html TEXT,
+    is_sensitive BOOLEAN NOT NULL,
+    is_required BOOLEAN NOT NULL,
+
+    UNIQUE (id, deliverable_type_id),
+    UNIQUE (module_id, position)
+);
+
+CREATE TABLE accelerator.deliverable_documents (
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables,
+    deliverable_type_id INTEGER NOT NULL REFERENCES accelerator.deliverable_types,
+    template_url TEXT,
+
+    CONSTRAINT deliverable_is_document CHECK (deliverable_type_id = 1),
+
+    FOREIGN KEY (deliverable_id, deliverable_type_id)
+        REFERENCES accelerator.deliverables (id, deliverable_type_id)
+);
+
+CREATE TABLE accelerator.project_deliverables (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables,
+    deliverable_status_id INTEGER NOT NULL REFERENCES accelerator.deliverable_statuses,
+    internal_comment TEXT,
+    feedback TEXT,
+
+    UNIQUE (project_id, deliverable_id)
+);
+
+CREATE INDEX ON accelerator.project_deliverables (deliverable_id);
+CREATE INDEX ON accelerator.project_deliverables (project_id);
+
+CREATE TABLE accelerator.project_documents (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    deliverable_id BIGINT NOT NULL REFERENCES accelerator.deliverables,
+    deliverable_type_id INTEGER NOT NULL REFERENCES accelerator.deliverable_types,
+    document_store_id INTEGER NOT NULL REFERENCES accelerator.document_stores,
+    created_by BIGINT NOT NULL REFERENCES users,
+    created_time TIMESTAMP WITH TIME ZONE NOT NULL,
+    modified_by BIGINT NOT NULL REFERENCES users,
+    modified_time TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
is this to change the description?

Reply:
It was for description changes, yeah, but since we're removing that ability, I've removed these columns.